### DISCUSSION TITLE
MQTT to OT message look up

### DISCUSSION THREAD

**Discussion by chezlong** [rvdbreemen/OTGW-firmware#237] [General]
Is there a cross reference between the MQTT messages and the OT message ID they are derived from? Had a sniff for keywords in the code to try to find a match (like "control_type_modulation" ) but drew a blank. 
[EDIT] closer inspection of the OT spec v2.2 I now see that it is bit 1 of message ID 3- slave configuration. It is called Control type [modulating, on/off]

**Comment by rvdbreemen** [comment]
Yes, it's all in the specification. And ofcourse if you can read source code, you will find out how I translate OT message ID's to useful MQTT messages.
